utils/byte_utils.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+func ConcatBytes(higher, lower byte) uint16 {
+	return (uint16(higher) << 8) | uint16(lower)
+}
+
+func GenerateData(bufferSize uint16) []byte {
```

Comment:
`GenerateData` is quite general. This method generates data in the `ping` command format.
Consider of renaming it :)